WordPress/src/main/java/org/wordpress/android/util/WPActivityUtils.java

```diff
@@ -15,4 +16,12 @@ public static Context getThemedContext(Context context) {
         }
         return context;
     }
+
+    public static String getNonNullString(TextView textView) {
+        if (textView != null && textView.getText().length() != 0) {
```

Comment:
We were checking `xxx.getText() != null` previously (There is no indication in the doc about `getText()` returning the null value, but I'm pretty sure we had the problem before and why we added the checks)